### PR TITLE
Simon rhinovirus (supersedes #66)

### PR DESCRIPTION
**Pull request by simonleandergrimm**
Supersedes #66 
Changes:
 - Edited representation of la population variables (` la_county_under_18_population` and ` la_county_over_18_population`
 - Edited explanation of Tecumseh study to include age of study.
 - Cleaned up `tecumseh_rhinovirus_infections_by_age`
 - Edited creation of dictionaries to have for loops before the iterated definition of variables (see `la_county_populations_by_age` as an example.)
 - Edited calculation of `la_age_groups` to be done through the creation of one dictionary instead of two steps. Not very happy with the aesthetics, @jeffkaufman , happy to get your suggestion for doing this in a prettier way.
 - Edited final comment on Rhinovirus transmission not being affected much by COVID-19.

### CODE REVIEW

File: pathogens/rhinovirus.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+la_county_adult_population = Population(
+    # 0.789 = the proportion of LA County residents over 18
+    people=0.789 * 9_818_605,
+    date="2020",
+    country="United States",
+    state="California",
+    county="Los Angeles County",
+    source="https://www.census.gov/quickfacts/fact/table/losangelescountycalifornia/AGE135221#AGE135221",
+)
```
Comment (jeffkaufman) [inline pathogens/rhinovirus.py:45]: Can we do the same approach here I suggested at https://github.com/naobservatory/p2ra/pull/123#discussion_r1213552516 ?

File: pathogens/rhinovirus.py [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+la_age_groups = {
+    "0-4": 285_140 + 273_131,
+    "5-19": (306_835 + 291_053) + (320_666 + 303_381) + (317_657 + 306_849),
+    "20-39": (330_183 + 327_625)
+    + (406_008 + 399_410)
+    + (413_566 + 394_103)
+    + (370_948 + 355_002),
+    "40+": 4_965_022  # subtracting <40yo population from total population
```
Comment (jeffkaufman) [inline pathogens/rhinovirus.py:92]: I'd delete the 40+ entry; see rest of comment below.

File: pathogens/rhinovirus.py [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+        people=n_people,
+        date="2020",
+        country="United States",
+        state="California",
+        county="Los Angeles County",
+        source=LA_DEMOGRAPHIC_DATA_2020,
+    )
+    la_county_populations_by_age[age_group] = age_group_population
+
+rhinovirus_1970s_tecumseh_based_la_estimate = (
```
Comment (jeffkaufman) [inline pathogens/rhinovirus.py:127]: Comment that we're assuming that people still get the same number of annual rhinoviruses for their age as they did in Tecumseh in the 70s, and adjusting it to represent the difference in age breakdown between the two contexts

**Review by jeffkaufman — CHANGES_REQUESTED** [review]

File: pathogens/rhinovirus.py (hunk 3 above)
Comment (simonleandergrimm) [reply pathogens/rhinovirus.py:127]: Added

File: pathogens/rhinovirus.py [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+    age_group_population = Population(
+        people=n_people,
+        date="2020",
+        country="United States",
+        state="California",
+        county="Los Angeles County",
+        source=LA_DEMOGRAPHIC_DATA_2020,
+    )
+    la_county_populations_by_age[age_group] = age_group_population
+
```
[the remark this answers is not in the document]
Comment (simonleandergrimm) [reply pathogens/rhinovirus.py:126]: done

File: pathogens/rhinovirus.py (hunk 1 above)
Comment (simonleandergrimm) [reply pathogens/rhinovirus.py:45]: Done

File: pathogens/rhinovirus.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (simonleandergrimm) [reply pathogens/rhinovirus.py:126]: Doing this i currently get the following error. I will look more into this later today or on Monday.
```
Traceback (most recent call last):
  File "/Users/simongrimm/code/p2ra/summarize.py", line 5, in <module>
    import pathogens
  File "/Users/simongrimm/code/p2ra/pathogens/__init__.py", line 12, in <module>
    pathogens[pathogen_name] = importlib.import_module(
  File "/opt/homebrew/Cellar/python@3.10/3.10.8/Frameworks/Python.framework/Versions/3.10/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/Users/simongrimm/code/p2ra/pathogens/rhinovirus.py", line 105, in <module>
    ) - sum(la_county_populations_by_age.values())
TypeError: unsupported operand type(s) for +: 'int' and 'Population'
```

[the remark this answers is not in the document]
Comment (simonleandergrimm) [reply pathogens/rhinovirus.py:126]: @simonleandergrimm for my own awareness

[the remark this answers is not in the document]
Comment (simonleandergrimm) [reply pathogens/rhinovirus.py:126]: Fixed this by adding an `__radd__` method to `Population`. I now run into an issue with `mypy`, @jeffkaufman happy to look at this on Monday.

Running mypy to check types...
pathogen_properties.py:466: error: Name "Number" already defined on line 301  [no-redef]
    class Number(Variable):
    ^
pathogens/rhinovirus.py:105: error: No overload variant of "sum" matches argument types "dict_values[str, Population]",
"Number"  [call-overload]
    ) - sum(la_county_populations_by_age.values(), start=Number(number=0))
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
pathogens/rhinovirus.py:105: note: Possible overload variants:
pathogens/rhinovirus.py:105: note:     def sum(Iterable[bool], /, start: int = ...) -> int
pathogens/rhinovirus.py:105: note:     def [_SupportsSumNoDefaultT <: _SupportsSumWithNoDefaultGiven] sum(Iterable[_SupportsSumNoDefaultT], /) -> Union[_SupportsSumNoDefaultT, Literal[0]]
pathogens/rhinovirus.py:105: note:     def [_AddableT1 <: SupportsAdd[Any, Any], _AddableT2 <: SupportsAdd[Any, Any]] sum(Iterable[_AddableT1], /, start: _AddableT2) -> Union[_AddableT1, _AddableT2]
Found 2 errors in 2 files (checked 24 source files)
FAIL: types

[the remark this answers is not in the document]
Comment (simonleandergrimm) [reply pathogens/rhinovirus.py:102]: *sum() likely trips up due to me defining start as `Number(number=0)`.

**Pull request merged**